dashboard/modules/job/job_manager.py

```diff
@@ -224,6 +225,13 @@ def _exec_entrypoint(self, logs_path: str) -> subprocess.Popen:
                 start_new_session=True,
                 stdout=logs_file,
                 stderr=subprocess.STDOUT,
+                # Ray intentionally blocks SIGINT in all processes, so if the user wants
```

Comment:
I found [this](https://github.com/ray-project/ray/blob/e4f9b3b7d951cdec2df11815ecb4ddf92a0a5afa/python/ray/services.py#L444-L451), though not sure if this implies any negative consequences. Maybe @edoakes can provide more insight?

Reply:
I see, looks like this only affects the case where Ray is started from an interactive terminal. Regardless, it only seems to affect Ray processes, so if I'm understanding things correctly it should be fine in our case, because the unblocking only affects the job entrypoint script and that shouldn't start any Ray processes as child processes. (Actors and tasks are Ray processes, but those are started by the raylet which is a separate process which is already running)

Reply:
Makes sense, thanks for raising the concern and explaining!